app/helpers/spreadsheet.rb

```diff
@@ -0,0 +1,41 @@
+require "google_drive"
+
+# Creates a session. This will prompt the credential via command line for the
+# first time and save it to config.json file for later usages.
+# See this document to learn how to create config.json:
+# https://github.com/gimite/google-drive-ruby/blob/master/doc/authorization.md
+
+# file = File.read('./sa.json')
+
+# session = GoogleDrive::Session.from_service_account_key(parsed_json)
+# First worksheet of
+# https://docs.google.com/spreadsheet/ccc?key=pz7XtlQC-PYx-jrVMJErTcg
+# Or https://docs.google.com/a/someone.com/spreadsheets/d/pz7XtlQC-PYx-jrVMJErTcg/edit?usp=drive_web
+# ws = session.file_by_id("1RFbedwlWR2I0d0ELllCeha9QCdZK_PQyJt_RcaerexU").worksheets[0]
+
+
+
+module LiveStats
+  def self.pull
+    $google_config = ENV['GOOGLEDRIVE_SECRET']
```

Comment:
is this the same kind of credentials as used by GCP? If so I've been using `GOOGLE_APPLICATION_CREDENTIALS` which the GCP plugin expects